tests/unit9.test.js

```diff
@@ -0,0 +1,60 @@
+const { test, expect } = require('@playwright/test');
+
+test('Perform Login:', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+  await page.getByPlaceholder('Username').fill('standard_user');
```

Comment:
Let's move user credentials to the separate module and import data from this module